src/Shared/Recruit.Vacancies.Client/Infrastructure/ReferenceData/ApprenticeshipProgrammes/ApprenticeshipProgrammesUpdateService.cs

```diff
@@ -39,8 +43,8 @@ public async Task UpdateApprenticeshipProgrammesAsync()
             {
                 Task.WaitAll(tasks.ToArray());
 
-                var standardsFromApi = standardsTask.Result.ToList();
-                var frameworksFromApi = frameworksTask.Result.ToList();
+                var standardsFromApi = standardsTask.Result.Distinct().ToList();
```

Comment:
the distinct calls here conflict with the other distinct call on insertion.

Reply:
removed it.